charts/ceph-csi-rbd/templates/provisioner-service.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if semverCompare "<=1.13" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
need to check both HTTP and grpc are enabled or not, if not just create a dummy service?

Reply:
This is needed, a StatefulSet isn't valid without a service. See this discussion from earlier:
https://github.com/ceph/ceph-csi/issues/81